src/pybind/mgr/dashboard_v2/module_standby.py

```diff
@@ -0,0 +1,62 @@
+import cherrypy
+import logging
+logger = logging.getLogger("dashboard")
+from mgr_module import MgrStandbyModule
+
+
+class StandbyModule(MgrStandbyModule):
+    def serve(self):
+        server_addr = self.get_localized_config('server_addr', '::')
+        server_port = self.get_localized_config('server_port', '7000')
+        if server_addr is None:
+            msg = 'no server_addr configured; try "ceph config-key set ' \
+                  'mgr/dashboard/server_addr <ip>"'
+            raise RuntimeError(msg)
+        logger.info("server_addr: %s server_port: %s",
+                    server_addr, server_port)
+        cherrypy.config.update({
+            'server.socket_host': server_addr,
+            'server.socket_port': int(server_port),
+            'engine.autoreload.on': False
+        })
+
+        module = self
+
+        class Root(object):
+            @cherrypy.expose
+            def index(self):
+                active_uri = module.get_active_uri()
+                if active_uri:
+                    logger.info("Redirecting to active '%s'", active_uri)
+                    raise cherrypy.HTTPRedirect(active_uri)
+                else:
+                    template = """
+                <html>
+                    <!-- Note: this is only displayed when the standby
+                         does not know an active URI to redirect to, otherwise
+                         a simple redirect is returned instead -->
+                    <head>
+                        <title>Ceph</title>
+                        <meta http-equiv="refresh" content="{delay}">
+                    </head>
+                    <body>
+                        No active ceph-mgr instance is currently running
+                        the dashboard.  A failover may be in progress.
```

Comment:
s/dashboard.  A failover/dashboard. A failover/

Reply:
In [1]: 'dashboard. A failover' == 'dashboard. A failover'
Out[1]: True

Reply:
two spaces:. s/dashboard.&nbsp;&nbsp;A failover/dashboard. A failover/

Reply:
Making two spaces after a period is a style, not a mistake and makes sense for editors like Vim or Emacs, where you can conveniently jump over sentences. It doesn't matter for the resulting HTML files. Also, this is the original text, so I'd like to keep it.